LanguageFeatures/Parts-with-imports/import_inheritance_A01_t01_part2.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Import inheritance is a only suggestion: Aka. other files’
+/// imports cannot break your code (at least if you’re not depending on them). A
+/// part file is never restricted by the imports it inherits from its parent
+/// file. It can ignore and override all of them with imports of its own. That
+/// allows a file, like a macro generated file, to import all its own
+/// dependencies and be completely self-contained when it comes to imports.
+///
+/// @description Check that for the part file import, inherited from the parent,
+/// can be overridden by it's own import.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=enhanced-parts
+
+import 'import_inheritance_A01_t01_lib2.dart';
+part of 'import_inheritance_A01_t01_part1.dart';
+
+void testPart2() {
+  Expect.equals("libGetter lib2", libGetter);
+  Expect.equals("LibMixin lib2", LibMixin.id);
+  Expect.equals("foo lib2", foo());
+  Expect.equals("C lib2", C.id);
```

Comment:
We should get `libVar` and `LibClass` from '...lib1.dart' and nothing from 'parts_lib.dart', but it would probably be useful to remove the declaration of `libVar` from '...lib1.dart' such that we get to test that (1) part2 can access names that are imported into part2 directly, and (2) part2 can access names that are imported by a parent and a part, namely '...part1.dart', and (3) part2 can access names that are imported by an indirect parent.

Reply:
I simply added `Expect.equals("libFunc", libFunc());`. `libFunc()` is not shadowed nor in `...lib1` nor in `...lib2`.